SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security
+
+Target takes the security of our software products and services seriously, which includes all source code repositories managed through [our GitHub organizations](https://opensource.target.com).
+
+If you believe you have found a security vulnerability in any Target-owned repository that meets Target's definition of a security vulnerability, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them to the Target Cyber Security team at _____.
```

Comment:
I think we should consider striking this. The reference material has an online form to fill out, and to my knowledge we have no such thing.

Reply:
Which reference material do you mean?

Reply:
Ryan's message, in which our policy was inspired from Microsoft's policy